### PR TITLE
Chore/changelog v2

### PR DESCRIPTION
**Pull request by saltcod**
Switches the source of https://supabase.com/changelog
**from:**  https://github.com/supabase/supabase/releases
**to**:  https://github.com/orgs/supabase/discussions/categories/changelog

— grab from discussions graphql endpoint instead of from releases endpoint in the restapi
— blends the old releases posts with the new discussions posts into one stream
— paginate the responses (not totally necessary now but if we add several changelogs a week (hopefully we will), we'll need this eventually) 

— 

Before merging, need to de-duplicate the content between these two sources. 
Will do this after PR approval, and before PR merge.

### CODE REVIEW

**Comment by MildTomato** [comment]
missing a env var ?

Screenshot 2023-11-08 at 7 46 53 PM

**Comment by alaister** [comment]
@saltcod currently getting a 500 error when trying to visit https://zone-www-dot-com-git-chore-changelog-v2-supabase.vercel.app/changelog

**Comment by saltcod** [comment]
thanks @alaister — a Discussion had an  tag that needs to be self-closing . 
Updated to accommodate these.

File: apps/www/pages/changelog.tsx [hunk 1]
```diff
@@ -6,40 +6,219 @@ import CTABanner from '~/components/CTABanner'
...
+              id
+              publishedAt
+              createdAt
+              url
+              title
+              # labels(first: 10) {
+                # nodes {
+                  # name
+                # }
+              # }
```
Comment (alaister) [inline apps/www/pages/changelog.tsx:118]: Any reason this still needs to be here if it's commented out?

**Review by alaister — APPROVED** [review]
LGTM! 🔥

File: apps/www/pages/changelog.tsx (hunk 1 above)
Comment (saltcod) [reply apps/www/pages/changelog.tsx:118]: aah right right was testing out a thing with that, will remove

**Pull request merged**